modules/math/src/geometry/primitives/modes.ts

```diff
@@ -16,13 +17,14 @@ export function getPrimitiveModeType(mode) {
     case GL.TRIANGLE_FAN: // draw a connected group of triangles.
       return GL.TRIANGLES;
     default:
-      // @ts-ignore
       return assert(false);
```

Comment:
This throws an exception, but typescript doesn't know that. If you replace this with `throw new Error('Unknown primitive mode`);` the typing will work.